### ISSUE TITLE
Google to Outlook sync creates new events at every sync

### ISSUE THREAD

**Issue by Nojwe** [phw198/OutlookGoogleCalendarSync#1512]
> Checklist:-
> * [X] [Upgrade](https://github.com/phw198/OutlookGoogleCalendarSync/wiki/Upgrading) to the latest release (alpha if available) ![latest release](https://img.shields.io/github/release/phw198/OutlookGoogleCalendarSync/all.svg)
> * [X] Reproduce the issue
> * [X] Complete the information below
> * [X] Provide your `OGcalsync.log` file (see [how-to](https://github.com/phw198/OutlookGoogleCalendarSync/wiki/Reporting-problems#locating-your-ogcalsynclog-file))

**OGCS Version**: v2.9.5.0
**Edition**: Installed
**Sync Direction**: G->O

**Bug Description**  
Every sync results in events being duplicated in Outlook. 

**Steps to Reproduce the Issue**  
1.Set up a Google to Outlook sync
2. Run sync

**Any other information**  
I've tried with these settings on and off individually, but always get the same behavior:
-Merge with existing entries
-Disable deletions
-Confirm deletions

Other settings for this profile (How tab)
Word obfuscation is on
Set privacy to Private
Set availability to Busy
Set category

What tab
-No attributes included
[OGcalsync.log](https://github.com/phw198/OutlookGoogleCalendarSync/files/9161627/OGcalsync.log)

**Comment by bluemantwo** [comment]
I can confirm the same thing.   I get additional copies of appointments every time it syncs.  Tried 2.9.0 and 2.9.5 same result.  Outlook 2010.  Windows 10.

**Comment by phw198** [comment]
@Nojwe It's actually telling you the problem:

> Due to your OGCS Outlook settings, all Outlook items have been filtered out!

Try not filtering out everything with the category exclusion option.

@bluemantwo No clue without your logfile.

**Comment by bluemantwo** [comment]
Thanks for helping look into this.  Here is my log file when I do a sync.  All 4 items it adds already exist, so I get duplicates.  Every time I sync, it adds them again. 

The log shows that no Outlook events are found, so it for some reason does not see existing events in my Outlook

EDIT: OK, nevermind, I found the problem.  None of my Outlook events were being found because I was using the default Regional Date Format.  I set it to FULL MONTH and now it works!  

So maybe Outlook 2010 (what I am using) needs this settings.  But in any case, it is now working perfectly!!!  Thanks (donation made via PayPal).

[OGcalsync.log](https://github.com/phw198/OutlookGoogleCalendarSync/files/9440320/OGcalsync.log)
